search/ui/js/search-widget.js

```diff
@@ -0,0 +1,47 @@
+var vipSearchModule = function () {
```

Comment:
We should use the modern JS syntax if possible. Not a big deal though.

Reply:
I agree. Again, didn't want to diverge from the Jetpack implementation at the beginning, so I think it's fine to leave it as is.